src/python/tensorflow_cloud/core/containerize.py

```diff
@@ -390,7 +392,12 @@ def get_docker_image(
             "Building and publishing docker image using Google "
             "Cloud Build: {}".format(image_uri)
         )
-        build_service = discovery.build("cloudbuild", "v1", cache_discovery=False)
+        build_service = discovery.build(
+            "cloudbuild",
+            "v1",
+            cache_discovery=False,
+            requestBuilder=google_api_client.TFCloudHttpRequest,
```

Comment:
did we cover this by unit test?

(no worries if it isn't covered. It can be follow-on CL/PR)

Reply:
Good catch. Added a detailed unit test for cloud build which includes this user agent testing.